Sources/Utility/ArgumentParser.swift

```diff
@@ -113,6 +113,21 @@ extension StringEnumArgument {
 }
 
 
+/// An argument representing a path (file / directory).
+///
+/// The path is resolved in the current working directory.
+public struct PathArgument: ArgumentKind {
+    public let path: AbsolutePath
+
+    public init?(arg: String) {
+        path = AbsolutePath(arg, relativeTo: currentWorkingDirectory)
+    }
+
+    public init(parser: inout ArgumentParserProtocol) throws {
+        path = AbsolutePath(try parser.associatedArgumentValue ?? parser.next(), relativeTo: currentWorkingDirectory)
+    }
```

Comment:
This will fail if arguments are not representable as relative paths. We should ensure validity of the path before passing it to `AbsolutePath`